test/unit/utils.spec.ts

```diff
@@ -18,7 +18,7 @@ describe("buildCursor", () => {
             name: "John Doe",
         };
 
-        const sort = {
+        const sort: Sort = {
             createdAt: 1,
```

Comment:
Question: Can we use `SortDirection` instead of the hardcoded integers?

Reply:
`SortDirection` isn't an enum :p It's a direct union of values such as `1`, `-1`, `asc`, etc.

Reply:
Makes sense!